modules/database/src/ioc/db/dbAccess.c

```diff
@@ -625,6 +625,8 @@ long dbEntryToAddr(const DBENTRY *pdbentry, DBADDR *paddr)
     paddr->field_size  = pflddes->size;
     paddr->special     = pflddes->special;
     paddr->dbr_field_type = mapDBFToDBR[dbfType];
+    paddr->ro = 0;
```

Comment:
The distinction is finer than "Virtual Only".  This can be accomplished with `DBF_NOACCESS`.  The `ro` flag disallows `dbPut()` when `dbr!=DBR_VFIELD`.  `dbGet()` is still allowed.

> ... do you expect all accesses to these to go through the put/get_vfield() methods?

Excepting device support, yes.  I see the present habit of going around `dbPut()`/`dbGet()` as a symptom of an interface which has grown inadequate.